Apps/frontend/src/const.js

```diff
@@ -4,3 +4,7 @@ export const CANVAS_WIDTH = 1000;
 export const CANVAS_HEIGHT = 500;
 export const NUM_INTERVALS_X = 20;
 export const NUM_INTERVALS_Y = 10;
+
+export const NUM_CHANNELS = 10;
+export const MIN_SWEEP  = 0.5;// <= 1
```

Comment:
Ah ok, I see that min and max are configurable, but why does the slider then have a range of 0-10. Couldn't they also take their values from here?

Reply:
The MIN_SWEEP and MAX_SWEEP correspond to the factor. So MAX_SWEEP=2 says that the signal is twice as fast if the slider is on the right. And if MIN_SWEEP is 0.5, then the signal is haft as fast. Hence, this is not equally distributed and is rescaled.